target/esp32/ll_cam.c

```diff
@@ -37,7 +37,7 @@ static inline int gpio_ll_get_level(gpio_dev_t *hw, int gpio_num)
 #if (ESP_IDF_VERSION_MAJOR >= 5)
 #define GPIO_PIN_INTR_POSEDGE GPIO_INTR_POSEDGE
 #define GPIO_PIN_INTR_NEGEDGE GPIO_INTR_NEGEDGE
-#define gpio_matrix_in(a,b,c) gpio_iomux_in(a,b)
+#include "rom/gpio.h"
```

Comment:
The recommended change is to:

1. include "esp_rom_gpio.h"
2. use `esp_rom_gpio_connect_in_signal` instead of `gpio_matrix_in`

This should work for release/v4.3 or later.

Reply:
what would the proper action be to not break support for older versions? if lower than 4.3, do not include esp_rom_gpio and define esp_rom_gpio_connect_in_signal = gpio_matrix_in, then use esp_rom_gpio_connect_in_signal in the code? Or do you have better idea?